### ISSUE TITLE
Numbers ending in "0" are "spell-out" incorrectly.

### ISSUE THREAD

**Issue by sagar-more** [mujib2953/SpellOut#1]
##### Description
Numbers that are ending in **0** are _***spelled-out***_ incorrectly.

#### Code:
```javascript
const {SpellOut} = require('spell-out');
SpellOut(12345630)
```

#### Expected Output:
```javascript
One Crore Twenty Three Lakh Fourty Five Thousand Six Hundred Thirty
```

#### Actual Output:
```javascript
One Crore Twenty Three Lakh Fourty Five Thousand Six Hundred Thirty undefined  
```

**Comment by mujib2953** [comment]
Thanks @sagar-more for pointing the issue, we have released version `0.0.3` and fixed this in the  same version.

Happy coding!!!

**Comment by mujib2953** [comment]
Closing this issue since it is fixed in [commit.](https://github.com/mujib2953/SpellOut/commit/7b9dcb766c4bd56f2db85787e17611b29d5d7653)
